packaging/rpm/ondemand.spec

```diff
@@ -340,6 +341,8 @@ touch %{_localstatedir}/www/ood/apps/sys/myjobs/tmp/restart.txt
 %files -n %{name}-selinux
 %{_datadir}/selinux/packages/%{name}-selinux/%{name}-selinux.pp
 
+%license /opt/ood/LICENSE.txt
```

Comment:
I think this should go under main `%files` as right now it's after the SELinux sub-package.  It may not matter, but might.